### ISSUE TITLE
Transition To AzureLib instead of GeckoLib

### ISSUE THREAD

**Issue by rainysux** [mariumbacchus/Soulslike-Weaponry#69]
AzureLib is a fork of gecko lib with a more of a focus on compatibility. It will be able for forge to be able to use this mod via connector. Its only a few changes to transition to Azure and I hope you will consider it!

**Comment by Mqony** [comment]
I agree, mod still doesn't work on connector and it would be amazing to use it on Forge. It would also make it easier for the developer as they wouldn't have to port to Forge 1.20.1

**Comment by mariumbacchus** [comment]
Actually might consider this when finally updating to 1.21.
